src/devices/Card/Ndef/UriType.cs

```diff
@@ -0,0 +1,235 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Runtime.Serialization;
+using System.Text;
+
+namespace Iot.Device.Ndef
+{
+    /// <summary>
+    /// Uri type
+    /// </summary>
+    public enum UriType
+    {
+        /// <summary>
+        /// No prepending is done, and the URI field
+        /// contains the unabridged URI.
+        /// </summary>
+        [Description("")]
```

Comment:
are you using Description anywhere? (haven't read the whole thing yet) If not I'd suggest to put the description in the comment

Reply:
Yes, I do use the Description to generate fully and properly the URL. So it's actually needed. See the enum helper for this.